python/dgl/mock_sparse2/elementwise_op_sp.py

```diff
@@ -1,31 +1,27 @@
 """DGL elementwise operators for sparse matrix module."""
-from typing import Union
 import torch
 
-from .diag_matrix import DiagMatrix
 from .sparse_matrix import SparseMatrix
 
 __all__ = ["sp_add"]
 
 
 def spsp_add(A, B):
-    """ Invoke C++ sparse library for addition """
+    """Invoke C++ sparse library for addition"""
     return SparseMatrix(
         torch.ops.dgl_sparse.spsp_add(A.c_sparse_matrix, B.c_sparse_matrix)
     )
 
 
-def sp_add(
-    A: Union[SparseMatrix, DiagMatrix], B: Union[SparseMatrix, DiagMatrix]
-) -> SparseMatrix:
+def sp_add(A: SparseMatrix, B: SparseMatrix) -> SparseMatrix:
```

Comment:
Do we support addition between sparse matrices with different sparsities for the 12/30 milestone?